<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>products</class>
 <widget class="QDialog" name="products">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Catalog&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>169</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="searchbutton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="gotocart">
      <property name="text">
       <string>Cart</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="nextPage">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="productsCart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Added to Cart</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="prod1">
      <property name="text">
       <string>Prod1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="product1">
      <property name="text">
       <string>Add to Cart</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="na1">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Product not available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>107</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="prod2">
      <property name="text">
       <string>Prod2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="product2">
      <property name="text">
       <string>Add to Cart</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="na2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Product not available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>154</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="prod3">
      <property name="text">
       <string>Prod3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="product3">
      <property name="text">
       <string>Add to Cart</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="na3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Product not available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>197</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="prod4">
      <property name="text">
       <string>Prod4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="product4">
      <property name="text">
       <string>Add to Cart</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="na4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Product not available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
